concent_api/utils/logging.py

```diff
@@ -176,34 +199,58 @@ def log_changes_in_subtask_states(client_public_key: bytes, count: int):
     ))
 
 
-def log_change_subtask_state_name(old_state, new_state):
+def log_change_subtask_state_name(
+    logger,
+    old_state,
+    new_state
+):
     logger.info('Subtask changed its state from {} to {}'.format(
         old_state,
         new_state,
     ))
 
 
-def log_new_pending_response(response_type: str, queue_name: str, task_id: str, subtask_id: str, client_public_key: bytes):
-    logger.info('New pending response in {} endpoint -- RESPONSE_TYPE: {} -- TASK_ID: {} -- SUBTASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
+def log_new_pending_response(
```

Comment:
why not using `replace_element_to_unavailable_instead_of_none` decorator here? Maybe it wolud be better to provide each of `task_id`, `subtask_id`, `requestor_public_key`, `provider_public_key` separately and not pass `Subtask` here?

Reply:
`log_new_pending_response` is called in `store_pending_message` and `store_pending_message` allows `subtask` to be `None`. That's why I did it like this, to avoid situation like `None.task_id`